src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```
@@ -432,6 +432,10 @@ services:
 
   PrestaShopBundle\Form\Admin\Sell\Catalog\FeatureType: ~
   PrestaShopBundle\Form\Admin\Sell\Catalog\FeatureValueType: ~
+  PrestaShopBundle\Form\Admin\Sell\Catalog\AttributeGroupType:
+    tags:
+      - { name: form.type }
```

```suggestion
  PrestaShopBundle\Form\Admin\Sell\Catalog\AttributeGroupType: ~
```
I don't think this is needed thanks to the `autoconfigure: true` set on the whole file SO we'd better keep the service definition as small as possible